### PR TITLE
Moved UnicodeFontHandler to background scripts

### LINKED ISSUE #6: Code style: Move UnicodeFontHandler to background scripts
@tdulcet you currently included the `UnicodeFontHandler` in the `/common` dir. Is there any reason for that?

Because actually it is only used in the `background` scripts, so it should also be in that place (`/modules` dir). Only really shared modules should be in the `common` dir.

### PR DESCRIPTION
**Pull request by tdulcet**
* Moved UnicodeFontHandler to background scripts. Closes #6.
* Minor update to README. Should have been in #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ See:
 
 ## Features
 * Puts your privacy first! Privacy is the default here.
-* Follows the [Firefox Photon Design](https://design.firefox.com/photon).
+* Follows the [Firefox](https://design.firefox.com/photon) and [Thunderbird](https://style.thunderbird.net/) Photon Design.
```
Comment (rugk) [inline README.md:33]: Oh nice, let’s hope these design systems don’t diverge too much… :upside_down_face:

File: src/background/modules/UnicodeFontHandler.js [hunk 2]
```diff
@@ -207,6 +207,7 @@ function applySettings(unicodeFont) {
  * @returns {void}
  */
 export async function init() {
+	// Remove once https://bugzilla.mozilla.org/show_bug.cgi?id=1595822 is fixed
```
Comment (rugk) [inline src/background/modules/UnicodeFontHandler.js:210]: Hmm, this comment comments a `await isMobile()` exclusion, but your bug is about Thunderbird?
How is this related?

`isMobile` AFAIK should only check (and in this case exclude) Firefox for Android…

File: src/options/options.html [hunk 3]
```diff
@@ -87,6 +87,7 @@ <h1 data-i18n="">Unicode autocorrection</h1>
 				</ul>
 			</section>
 
+			<!-- Remove "mobile-incompatible" once https://bugzilla.mozilla.org/show_bug.cgi?id=1595822 is fixed -->
```
Comment (rugk) [inline src/options/options.html:90]: Same here…
Do you “abuse” this mobile-incompatible blacklisting for Thunderbird? :upside_down_face: 

If so, that’s of course not the proper way to do so. Because this add-on may (as for the auto-correction part, at least), run fine on Firefox for Android (Fenix, if they allow all add-ons in a future version). So we need a different `thunderbird-incompatible` CSS and `isThunderbird` JS or so.

File: src/background/modules/UnicodeFontHandler.js (hunk 2 above)
Comment (tdulcet) [reply src/background/modules/UnicodeFontHandler.js:210]: No, that bug is about implementing the WebExtension context menu API in Firefox for Android.

File: src/options/options.html (hunk 3 above)
Comment (tdulcet) [reply src/options/options.html:90]: See above. The context menu does not currently work in the compose window of Thunderbird because of [bug 1656506](https://bugzilla.mozilla.org/show_bug.cgi?id=1656506), although it does not need to be disabled and it looks like this may be fixed by [bug 1670832](https://bugzilla.mozilla.org/show_bug.cgi?id=1670832).

**Review by rugk — APPROVED** [review]
Ah sorry about the comments, seems I was not fully awake when reviewing this. Of course your added comments are good and point out the FF for Android (Fenix) issues. (Though one might need to re-create these issues at https://github.com/mozilla-mobile/fenix/ so they see it. But they work on add-on compatibility in general anyway, already, so I see no need here.)
It seems I read Thunderbird and then applied everything I saw to Thunderbird :upside_down_face:  Sorry. It's all right, of course.
